ui_framework/doc_site/src/views/tool_bar/tool_bar_search_only.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+} from '../../../../components';
+
+export const ToolBarSearchOnly = () => (
+  <KuiToolBar className="kuiToolBar--searchOnly">
```

Comment:
> Maybe that's a problem and we shouldn't be showcasing an example that doesn't actually exist in our code currently

Great catch! I didn't realize it wasn't being used. Let's remove it.

> I see the benefit of this being if we ever want to change the css name, it's not a large refactor, just change it in one place, and it's also more readable - isSearchOnly vs the css name.

Yeah, that's exactly what I was thinking too. I would like to treat className as an escape hatch, which engineers can use to hack together a solution that's not provided by the UI Framework.

 To @kjbekkelund's point, I think the fact that this component requires a class like `kuiToolBar--searchOnly` means that I designed it poorly to begin with. So if we were to keep it, I would probably want to refactor the component to make this class unnecessary.